client/landing/jetpack-cloud/sections/backups/style.scss

```
@@ -20,3 +20,11 @@
 .backups__page .main {
 	margin-left: auto;
 }
+
+.backups__is-loading {
+	margin: 2rem 1rem;
+	height: 245px;
+	background-color: var( --color-neutral-10 );
+	animation: loading-fade 1.6s ease-in-out infinite;
```

this same effect can be achieved with 
```suggestion
	@include placeholder();
```

---

Good to know about this mixin. Thanks